be/fmt.h

```diff
@@ -200,15 +200,15 @@ struct m0_be_fmt_log_store_header {
 struct m0_be_fmt_group_cfg;
 
 struct m0_be_fmt_log_header {
-	uint64_t    flh_serial;
+	uint64_t    flh_serial_num;
 	m0_bindex_t flh_discarded;
 	m0_bindex_t flh_group_lsn;
```

Comment:
Please add comments for these field members.
Especially for "flh_serial_num" and "flh_group_lsn".
What are the differences?

Reply:
done.